porch/repository/pkg/git/git.go

```diff
@@ -124,6 +124,10 @@ type gitRepository struct {
 }
 
 func (r *gitRepository) ListPackageRevisions(ctx context.Context) ([]repository.PackageRevision, error) {
+	if err := r.update(ctx); err != nil {
```

Comment:
Should we rename `update`?  It's not self-evident what it does... (though admittedly I can't think what _else_ it could do!)

Reply:
done. I went with `fetchRemoteRepository`
(sorry, I'm apparently quite bad with naming)